CMakeLists.txt

```diff
@@ -295,7 +322,7 @@ endif()
 # Packaging
 ################################################################################################
 
-if (BUILD_PACKAGE)
+if (BUILD_PACKAGE AND (NOT WIN32))
```

Comment:
minor nit: what is the reason to check for `NOT WIN32` - was it failing something if you don't add it? I wonder if we could eventually add logic in here under the `if (BUILD_PACKAGE)` to produce a package, e.g. nuget or alike, even if we build on Windows. My selfish interest - is we'd likely need this CMake flow for our organization telemetry.